DemoApp/CreateChatViewController.swift

```diff
@@ -27,7 +27,7 @@ class CreateChatViewController: UIViewController {
                 super.createNewMessage(text: text)
                 
                 // Present the new chat and controller
-                let vc = ChatMessageListVC()
+                let vc = ChatChannelVC()
```

Comment:
I would recommend to start removing VC and any other abbreviation from names, but this looks like out of scope for this PR and should be discussed later

>https://github.com/raywenderlich/swift-style-guide#naming
>generally avoiding abbreviations

Reply:
The reason we have `VC` is that we had a lot of big names in the project, and so the team decided we would suffix the view controllers with `VC`. Anyways, we then further improved the naming, and they are not that big anymore, so we can revisit this later since changing it now will cause a lot of breaking changes to the customers (more than we are already causing here). But I also prefer not having abbreviations. But we can do this now if the team agrees @dmigach @tbarbugli @SagarSDagdu @b-onc